api/v2.go

```diff
@@ -53,6 +54,7 @@ func NewV2Endpoints(c *Client) *V2Endpoints {
 	v2 := &V2Endpoints{c,
 		&UserProfileService{c},
 		&AlertChannelsService{c},
+		&AlertProfilesService{c, AlertTemplatesService{c}},
```

Comment:
shouldn't we follow this pattern for nested services? https://github.com/lacework/go-sdk/blob/main/api/v2_vulnerabilities.go#L23-L36

Reply:
Is this not slightly different in the vulnerability services are at the same level. Templates is nested under profiles.

V2.Vulnerabilities.Hosts.Func() & V2.Vulnerabilities.Containers.Func()

vs 

V2.Vulnerabilities.AlertProfiles.Func()
V2.Vulnerabilities.AlertProfiles.Templates.Func(alertProfileID)

Reply:
I'll refactor as you've suggested and see.

V2.Vulnerabilities.Alert.Profiles.Func()
V2.Vulnerabilities.Alert.Templates.Func()

Not sure if it matters but we also V2.Vulnerabilities.AlertRules and V2.Vulnerabilities.AlertChannels then not nested under "Alert"

Reply:
Seeing it, I do prefer the suggested way 👍